charts/weblate/values.yaml

```diff
@@ -147,27 +147,29 @@ tolerations: []
 affinity: {}
 
 postgresql:
-  # postgresql.postgresqlUsername -- PostgreSQL user should be a superuser to
+  auth:
+  # PostgreSQL user should be a superuser to
   # be able to install pg_trgm extension. Alternatively you can install it
   # manually prior starting Weblate.
-  postgresqlUsername: postgres
-  postgresqlPassword: weblate
-  postgresqlDatabase: weblate
+    enablePostgresUser: true
+    postgresPassword: weblate
+    database: weblate
   service:
-    port: 5432
+    ports:
+      postgresql: 5432
   enabled: true
   # postgresql.postgresqlHost -- External postgres database endpoint, to be
   # used if `postgresql.enabled == false`
   # @default -- `None`
-  postgresqlHost:
```

Comment:
This is removed, but still used in the template.

Reply:
I added it